lesson_00/Kyvonreadme.md

```diff
@@ -0,0 +1,28 @@
+# Learn about Kyvon Thompson
```

Comment:
This is a decent start, but you should add something about yourself that will help other collaborate with you professionally (e.g. what are your best working hours? how do you prefer that teammates communicate with you?).